prow/plank/README.md

```diff
@@ -0,0 +1,41 @@
+# Plank
+
+Plank is the controller that manages the job execution and lifecycle for jobs running in k8s.
+
+### Usage
+```bash
+$ bazel run //prow/cmd/plank -- --help
+```
+
+### Configuration
+The following job template is based on the format for `spyglass`.
+
+```yaml
+# config.yaml
+
+plank:
+  allow_cancellations: true
+  job_url_prefix: 'https://<domain>/view/gcs/<bucket-name>'
+  job_url_template: 'https://<domain>/view/gcs/<bucket-name>/pr-logs/pull/{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
+  report_template: '[Full PR test history](https://<domain>/pr-history?org={{.Spec.Refs.Org}}&repo={{.Spec.Refs.Repo}}&pr={{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}})'
```

Comment:
```yaml
  report_template: ... # used to summarize test results when reporting to GItHub
```

maybe we should omit this if we want people to use `crier` moving forward? @krzyzacy

Reply:
what is `crier` if you don't mind me asking?

Reply:
It's a component of Prow that is in charge of reporting job status out to external sources

Reply:
ah, i see. it's an interface for reporting then. and you implement an adapter for the sources you want to report to.

Reply:
hummmm - crier uses whatever URL set by plank? (it's the difference between decorated && non-decorated?)